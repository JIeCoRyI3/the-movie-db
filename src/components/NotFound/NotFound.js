import React from 'react';
import { notFound } from './NotFound.module.css';

function NotFound() {
    return <div className={notFound}>NOT FOUND</div>;
}

export default NotFound;
