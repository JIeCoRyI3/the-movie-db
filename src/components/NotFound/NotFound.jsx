import React from 'react';
import styles from './NotFound.module.css';
import emoji from '../../assets/images/emoji_1.png'

function NotFound() {
    return (
        <div className={styles.container}>
          <h2 className={styles.text}>FILM NOT FOUND</h2>
          <div className={styles.emoji}>
            <img src = {emoji} alt={''}/>
          </div>
        </div>
    );
}

export default NotFound;
