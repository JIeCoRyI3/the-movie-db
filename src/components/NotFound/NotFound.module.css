
.container{
    display: flex;
    flex-direction: column-reverse;
    justify-content: flex-end;
    width: 100%;
    height: calc(100vh - 34em);
    background-color: #ebf6fc;
}
.emoji{
    height: 85%;
    background-image: url("../../assets/images/emoji_1.png");
    background-repeat: no-repeat;
    background-size: 30%;
    background-position: center;
}

.text{
    height: 15%;
    font-size: 4em;
    font-weight: bold;
    font-style: italic;
    text-align: center;
    margin: auto;
    line-height: 4em;
}
