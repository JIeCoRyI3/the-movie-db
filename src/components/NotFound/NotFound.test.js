import React from 'react';
import { shallow } from 'enzyme';
import NotFound from './NotFound';

describe('<NotFound/> render', () => {
    let wrapper;

    beforeEach(() => {
        wrapper = shallow(<NotFound />);
    });

    it('should render NotFound component', () => {
        expect(wrapper).toMatchSnapshot();
    });

    it('should be not empty text', () => {
        expect(wrapper.find('p')).not.toBe(null);
    });
});
