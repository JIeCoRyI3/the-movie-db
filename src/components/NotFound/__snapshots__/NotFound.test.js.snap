// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound/> render should render NotFound component 1`] = `
<div>
  <h2>
    FILM NOT FOUND
  </h2>
  <div>
    <img
      alt=""
      src="test-file-stub"
    />
  </div>
</div>
`;
